### DISCUSSION TITLE
Переробити список Middlewares в Environment Store на Set

### DISCUSSION THREAD

**Discussion by boredarthur** [Maks-Jago/SwiftUI-UDF#36] [General]
Зараз, може виникнути баг дублювання мідлварів, якщо не перевизначати EnvironmentStore при ініціалізації застосунку.
Треба зробити proof-check цього моменту і змінити звичайний Array мідлварів на OrderedSet

**Comment by kiddden** [comment]
Fixed in PR: https://github.com/Maks-Jago/SwiftUI-UDF/pull/48
